tests/zfs-tests/include/libtest.shlib

```diff
@@ -20,14 +20,13 @@
 #
 
 #
-# Copyright 2009 Sun Microsystems, Inc.  All rights reserved.
-# Use is subject to license terms.
-# Copyright (c) 2012, 2018 by Delphix. All rights reserved.
-# Copyright (c) 2017 by Tim Chase. All rights reserved.
-# Copyright (c) 2017 by Nexenta Systems, Inc. All rights reserved.
-# Copyright (c) 2017 Lawrence Livermore National Security, LLC.
-# Copyright (c) 2017 Datto Inc.
-# Copyright (c) 2017 Open-E, Inc. All Rights Reserved.
+# Copyright (c) 2009, Sun Microsystems Inc. All rights reserved.
+# Copyright (c) 2012, 2018, Delphix. All rights reserved.
+# Copyright (c) 2017, Tim Chase. All rights reserved.
+# Copyright (c) 2017, Nexenta Systems Inc. All rights reserved.
+# Copyright (c) 2017, Lawrence Livermore National Security LLC. All rights reserved.
+# Copyright (c) 2017, Datto Inc. All rights reserved.
+# Copyright (c) 2017, Open-E Inc. All rights reserved.
 # Use is subject to license terms.
```

Comment:
Here and throughout, please don't modify existing copyright notices.

Reply:
I didn't change the intention nor the meaning, just formating that was off.
There is no legal consequence.

Reply:
Companies I've worked for have alway advised me not to modify copyrights. I'm not an expert on copyright law, so I won't try to debate whether or not there is legal consequence.

The formatting you've done does change the meaning. By moving the "Use is subject..." line to the bottom, it no longer looks like it applies only to the Sun copyright line. Additionally, it's not uncommon for authors to use editor automation to update or add copyright lines. These changes could inconvenience those authors.

All of which is to say I don't think the pros of the reformat outweigh even the known cons.

Reply:
> Companies I've worked for have alway advised me not to modify copyrights

And I 100% agree with them. Just today I scolded another contributor for changing a licence on a file, without consent of the author.

> By moving the "Use is subject..." line to the bottom, it no longer looks like it applies only to the Sun copyright line.

No it does not:
"Use (of the file) is subject to (the CDDL) licence terms". Has nothing to do with sun or any of the other copyright holders directly. It does not say "is subject to Sun's copyright, or Sun's licence terms".

To be frankfully honest: Legally this is the biggest bullshit ever.... This is a legacy thing from decades ago, when it was required. Use is ALWAYS subject to licence terms and everyone (listed or not) automatically (by law) gains copyright since many years now.

The reason people keep doing it, is simply to keep track of why has rights where (for which this is a great tool!), but legally this is not needed at all.

> Additionally, it's not uncommon for authors to use editor automation to update or add copyright lines

That explains why people screw them up so badly at times. But I need to find the first repo where "my editor automation made me do it" was valued as a valid reason to screw up formating.


TL:DR
These are not "copyrights" these are references. Copyright is governed by (international) law.

Reply:
But okey...
I'll let @behlendorf as chief maintainer for the ZoL repo decide.

Reply:
For the authoritative copyright attribution we've always relied on the git commit history and signed-off-by.

Cosmetically cleaning up these lines isn't an issue.  It does feel a little out of scope for this PR, but aside from that I don't have any objection.

Reply:
Yeah I think @c0d3z3r0 was a bit overzealus... reformatting only the actually changed headers would be my prefference also. That being said: its a shame to revert anymore just because “little out of scope” agreed?